<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;方正粗黑宋简体&quot;;</string>
        </property>
        <property name="text">
         <string>姓     名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lE_r_sname"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;方正粗黑宋简体&quot;;</string>
        </property>
        <property name="text">
         <string>班     级：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lE_r_class"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;方正粗黑宋简体&quot;;</string>
        </property>
        <property name="text">
         <string>学     号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lE_r_sno"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;方正粗黑宋简体&quot;;</string>
        </property>
        <property name="text">
         <string>登录密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lE_r_pwd"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pBtn_r_enter">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;方正粗黑宋简体&quot;;</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_r_back">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;方正粗黑宋简体&quot;;</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/123/02.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;华文琥珀&quot;;</string>
   </property>
   <property name="text">
    <string>欢迎新用户进行注册！</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>lE_r_sname</zorder>
  <zorder>label_2</zorder>
  <zorder>lE_r_class</zorder>
  <zorder>label_3</zorder>
  <zorder>lE_r_sno</zorder>
  <zorder>label_4</zorder>
  <zorder>lE_r_pwd</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
